README.md

```diff
@@ -205,6 +205,11 @@ All modules involved in data import (mod-data-import, mod-source-record-manager,
 * `OKAPI_URL`
 * `ENV` (unique env ID)
 
+There are another important properties - `number of partitions` for topics `DI_INITIALIZATION_STARTED` and `DI_RAW_RECORDS_CHUNK_READ`
+which are created during tenant initialization, the values of which can be customized with
+ `DI_INITIALIZATION_STARTED_PARTITIONS` and `DI_RAW_RECORDS_CHUNK_READ_PARTITIONS` env variables respectively. 
+Default value - `1`.
```

Comment:
Is there an option in the Kafka wrapper to specify the Replication Factor (number of brokers to which the data is replicated)?

Reply:
hi, yes can be set via env var REPLICATION_FACTOR

Reply:
@okolawole-ebsco and @KaterynaSenchenko , should the Replication Factor be set during topic creation as well?

Reply:
according to what i have read so far, replication factor will be set to 1. I think that is the correct value for a default for Kafka.

Reply:
What I meant to ask is: should the RF be configurable (if it's not configurable already) so that hosting providers can set it despite its default value can be 1?

Reply:
yes, as mentioned earlier we have this env var REPLICATION_FACTOR, to be set value for topics created by this module, by default the value of RF is 1